compiler/main/driver.cpp

```diff
@@ -1054,6 +1055,7 @@ static ArgumentDescription arg_desc[] = {
  {"print-unused-functions", ' ', NULL, "[Don't] print the name and location of unused functions", "N", &fPrintUnusedFns, NULL, NULL},
  {"set", 's', "<name>[=<value>]", "Set config value", "S", NULL, NULL, readConfig},
  {"task-tracking", ' ', NULL, "Enable [disable] runtime task tracking", "N", &fEnableTaskTracking, "CHPL_TASK_TRACKING", NULL},
+ {"interleave-memory", ' ', NULL, "Enable [disable] memory interleaving", "N", &fEnableMemInterleaving, "CHPL_INTERLEAVE_MEM", NULL},
```

Comment:
Note that these flags are currently alphabetized within each section.

Reply:
Thanks, I changed the order

Reply:
Whoops, I actually had this in the wrong category. I thought this was under a section that would only appear with `--devel`, but it was just under a common flag called `DRIVER_ARG_DEVELOPER`. I'm moving this down to the `Developer Flags -- Miscellaneous:` section, which seems to be very inconsistently alphabetized.